core/controllers/cron.py

```diff
@@ -86,6 +87,14 @@ def get(self):
         email_services.send_mail_to_admin(email_subject, email_message)
 
 
+class AppendLastDashboardStatsToList(base.BaseHandler):
+    """Handler for appending dashboard stats to a list."""
+
+    @require_cron_or_superadmin
+    def get(self):
+        """Handles GET requests."""
+        user_jobs_one_off_test.DashboardStatsOneOffJobTests.run_one_off_job()
+
```

Comment:
I knew that these can be manually tested through the admin panel. But my attempts of testing this manually falied. It kept throwing a wierd error - 

> No module named webtest 

Not sure why this error was being thrown. I tried restarting the server a few times but it did not seem to go away. So I decided to commit without testing this line, and thus added a line comment here!